WordPress/Classes/ViewRelated/Me/My Profile/MyProfileHeaderView.swift

```diff
@@ -25,7 +25,7 @@ class MyProfileHeaderView: UITableViewHeaderFooterView {
     var gravatarEmail: String? = nil {
         didSet {
             if let email = gravatarEmail {
-                gravatarImageView.downloadGravatar(for: email, gravatarRating: .x)
+                downloadAvatar()
```

Comment:
It produces a new Xcode warning (email unused).

Reply:
Oh I missed this. Fixed. 👍